polyfill/test/parse.mjs

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env -S node --experimental-modules
+
+/*
+ ** Copyright (C) 2020 Igalia S.L. All rights reserved.
+ ** This code is governed by the license found in the LICENSE file.
+ */
+
+import Demitasse from '@pipobscure/demitasse';
+const { describe, it, report } = Demitasse;
+
+import Pretty from '@pipobscure/demitasse-pretty';
+const { reporter } = Pretty;
+
+import Assert from 'assert';
+const { ok: assert, strictEqual: equal, deepStrictEqual: dEqual, throws } = Assert;
```

Comment:
```suggestion
import { strict as assert } from 'assert';
const { ok: assert, equal, deepEqual: dEqual, throws } = assert;
```

Reply:
Sure.

Reply:
This was just copied over from other tests. I propose that we let this happen without this, and I'll file another issue for fixing this across the codebase.